src/map/map.c

```diff
@@ -5425,6 +5425,30 @@ static bool map_zone_mf_cache(int m, char *flag, char *params)
 			else if( map->list[m].flag.nocashshop )
 				map_zone_mf_cache_add(m,"nocashshop");
 		}
+	} else if(!strcmpi(flag, "nostorage")) {
+		if( !state ) {
+			if( map->list[m].flag.nostorage != 0 ) {
+				sprintf(rflag,"nostorage\t%d",map->list[m].flag.nostorage);
+				map_zone_mf_cache_add(m,rflag);
+			}
+		} if( sscanf(params, "%d", &state) == 1 ) {
+			if( state != map->list[m].flag.nostorage ) {
+				sprintf(rflag,"nostorage\t%s",params);
```

Comment:
params was already converted to int by previous scanf. probably better replace ``%s`` to ``%d`` and use state?
this need for avoid possible buffer overflow